<launch>

    <node pkg="nav" type="nav_node" name="nav" output="screen">
        <!-- set a private parameter for the node -->
        <param name="goalTopic" type="string" value="/move_base_simple/goal"/>
        <param name="velTopic" type="string" value="cmd_vel"/>
        <param name="odomTopic" type="string" value="/Odometry"/>
        <param name="static_map_url" type="string" value="$(find nav)/map/static_map.png"/>

        <param name="init_x" type="double" value="6.75"/>
        <param name="init_y" type="double" value="9.4"/>
        <!-- BLUE is 1, RED is 0 -->
        <param name="preset_side" type="int" value="0"/>
    </node>

</launch>
